GREEDY/LogHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace GREEDY
+{
+    public class LogHandler : DelegatingHandler
+    {
+        private static object lockObject = new object();
+        private string logEntrySeperator => "---------------------------------------------------";
+
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
```

Comment:
name is too ling. need to split or change names to shorter

Reply:
this is an override method, i can only change is names of the parameters